justfile

```
@@ -0,0 +1,90 @@
+targets := "arm-unknown-linux-gnueabi armv7-unknown-linux-gnueabihf x86_64-unknown-linux-gnu x86_64-unknown-linux-musl thumbv6m-none-eabi thumbv7em-none-eabi thumbv7em-none-eabihf thumbv7m-none-eabi"
+target_dir := "target"
+
+#----------
+# Building
+#----------
+
+build: check-formatting test test-all build-benches check-readme check-links
+
+# Build the benches
+build-benches:
+    cargo bench --no-run
+
+# Build the simulator
+build-simulator:
+    cd simulator; \
+    cargo build --release --no-default-features
+
+# Run cargo test in release mode
+test:
+    cargo test --release
+
+# Run cargo test in release mode with all features enabled
+test-all:
+    cargo test --release --all-features
+
+# Check the formatting
+check-formatting:
+    cargo fmt --all -- --check
+
+# Cross compiles embedded-graphics, tinytga and tinybmp for a target
+build-target target *args:
+    cargo build --target {{target}} {{args}}
+    cargo build --target {{target}} --all-features {{args}}
+
+# Cross compiles embedded-graphics, tinytga and tinybmp for all targets
+build-targets *args:
+    for target in {{targets}}; do just build-target $target {{args}}; done
```

I missed this when I fixed the for loops:
```suggestion
    #!/usr/bin/env bash
    set -e
    for target in {{targets}}; do just build-target $target {{args}}; done
```